scripts/qa_crowbarsetup.sh

```diff
@@ -2749,7 +2759,7 @@ function get_novacontroller()
     local element=`crowbar nova proposal show default | \
         rubyjsonparse "
                     puts j['deployment']['nova']\
-                        ['elements']['nova-multi-controller']"`
+                        ['elements']['`nova_role_prefix`-controller']"`
```

Comment:
:-1: test failed with qa_crowbarsetup.sh: command substitution: line 2769: unexpected EOF while looking for matching `"'